src/autopas/selectors/tuningStrategy/TuningStrategyFactory.cpp

```diff
@@ -60,14 +62,16 @@ std::unique_ptr<autopas::TuningStrategyInterface> autopas::TuningStrategyFactory
         fallbackCellSizeFactors = std::make_unique<autopas::NumberInterval<double>>(allowedCellSizeFactors.getMin(),
                                                                                     allowedCellSizeFactors.getMax());
       }
+      fallbackVerletRebuildFrequencies =
+          std::make_unique<autopas::NumberSetFinite<int>>(allowedVerletRebuildFrequencies.getAll());
```

Comment:
Please handle this like `cellSizeFactors`, where `NumberSetFinite` is only used if the set is finite, and if it is not `NumberInterval` is used.

Reply:
It is always finite. Therefore no need of the NumberInterval

Reply:
Currently, AutoPas does not appear to be able to handle unbounded sets, so yes allowed rebuild frequencies is always finite. But I would like to be able to handle unbounded sets in the future, in which case allowed rebuild frequencies could be infinite. 

This doesn't explicitly have to be done in your pull request, so if you would like to leave your code as it is, simply add a comment above this line saying that AutoPas assumes Verlet Rebuild Frequency is bounded so it is always finite.

Reply:
I didn't think about this case. You are right and I tried to change this, but at the moment there is no class available for a NumberSet with no Maximum. I think the person who likes to have this case should be able to create the class as wanted.
If you think oherwise I can create a new class for this.